aurelia-slickgrid/src/aurelia-slickgrid/sorters/sorterUtilities.ts

```diff
@@ -1,7 +1,8 @@
 import { FieldType } from './../models/fieldType.enum';
 import { Sorters } from './index';
+import { Column } from '../models/column.interface';
 
-export function sortByFieldType(value1: any, value2: any, fieldType: FieldType, sortDirection: number) {
+export function sortByFieldType(value1: any, value2: any, fieldType: FieldType, sortDirection: number, sortColumn: Column) {
```

Comment:
this utility is used in the `CollectionService` and should be modified accordingly in that class